src/Microsoft.Win32.Primitives/src/Configurations.props

```diff
@@ -4,6 +4,8 @@
     <BuildConfigurations>
       netcoreapp-Unix;
       netcoreapp-Windows_NT;
+      uap-Unix;
```

Comment:
what is uap-Unix for? Something to do with CoreRT (I thought that was going to be a different vertical)